### PR TITLE
allowing to override the streaming gimbal ip and port

### PR DESCRIPTION
**Pull request by frederictaillandier**

### CODE REVIEW

**Comment by frederictaillandier** [comment]
@Jaeyoung-Lim @julianoes  This should allow people to have another system to handle the gimbal and camera simulation, like for exemple a companion computer that would handle any kind of payload :)

File: src/gazebo_gimbal_controller_plugin.cpp [hunk 1]
```diff
@@ -348,7 +348,22 @@ void GimbalControllerPlugin::Load(physics::ModelPtr _model,
...
+  } else {
+    this->udp_gimbal_host_ip = "127.0.0.1";
+  }
 
+  if (this->sdf->HasElement("udp_gimbal_port_remote")) {
+    this->udp_gimbal_port_remote = _sdf->Get<int>("udp_gimbal_port_remote");
+  } else {
+    this->udp_gimbal_port_remote = 13030;
+  }
+  gzwarn << "[gazebo_gimbal_controller_plugin] Streaming gimbal mavlink stream to ip: " << this->udp_gimbal_host_ip  << " port: " << this->udp_gimbal_port_remote << std::endl;
```
Comment (julianoes) [inline src/gazebo_gimbal_controller_plugin.cpp:366]: Should that be a warning or just info?

**Review by julianoes — COMMENTED** [review]
I think that's correct and it's understandable :+1:

File: src/gazebo_gimbal_controller_plugin.cpp (hunk 1 above)
Comment (frederictaillandier) [reply src/gazebo_gimbal_controller_plugin.cpp:366]: to me it makes sense for it to be an info, but I just wanted to mimic the way it has been done there : 
https://github.com/PX4/PX4-SITL_gazebo/blob/6c3f1f8af6d062e4225a152a6d2a7bc5e276f537/src/gazebo_gst_camera_plugin.cpp#L232

Comment (julianoes) [reply src/gazebo_gimbal_controller_plugin.cpp:366]: Alright sure.

**Review by julianoes — APPROVED** [review]

**Pull request merged**